teuthology/orchestra/run.py

```diff
@@ -9,13 +9,13 @@
 import gevent
 import gevent.event
 import socket
-import pipes
 import logging
 import shutil
 
 from teuthology.contextutil import safe_while
 from teuthology.exceptions import (CommandCrashedError, CommandFailedError,
                                    ConnectionLostError)
+from teuthology.orchestra.run_helper import Raw, quote
```

Comment:
Instead of `run_helper` I suggest more appropriate name for the module, `util` or `tool` or `common` or for example

Reply:
I find run_helper appropriate since it contains classes that help run.py.

BTW I moving rest of misc. classes from run.py to run_helper.py too.

Reply:
that looks weird, if you really want to have run based classes it would have more appropriate to have this layout:

teuthology/orchestra/run/__init__.py
teuthology/orchestra/run/helper.py

instead of underscore separated.

Reply:
I personally feel this a bit of overkill but I don't really mind.

@batrick Any opinion?